### ISSUE TITLE
Assign names to error codes

### ISSUE THREAD

**Issue by sajattack** [pspdev/psp-ghidra-scripts#9]
I'm often finding myself cross-referencing my ghidra output with an error list like [this one](https://github.com/uofw/uofw/blob/f099b78dc0937df4e7346e2e417b63f471f8a3af/include/common/errors.h). We could define these as named variables in the script I think, similar to what the SVD plugin does to assign names to hardware registers. Would save a lot of time.

**Comment by artart78** [comment]
@sajattack are data type archives good enough for this or do you have other ideas? (This still requires pressing E and choosing the correct value, idk if we could do it automatically and if it would be a good idea?)

**Comment by sajattack** [comment]
Yeah I suppose it would be pretty error-prone to try to do automatically. With the datatype, can I just pick the enumeration and have it determine the variant? I don't know how it works. Maybe a script would be another good option?
